src/routes/capacity.js

```diff
@@ -181,7 +149,6 @@ router.get(
   asyncRoute(async (req, res) => {
     const productId = Number(req.params.productId);
     const periodQuery = Number(req.query.period) || 30;
-    const withPools = req.query.withPools === 'true';
```

Comment:
withPools as query param is now dropped. for `/v2/capacity/{productId}` `capacityPerPool` field will always be included